apps/condo/domains/common/utils/text.spec.js

```diff
@@ -0,0 +1,41 @@
+const { normalizeText } = require('./text')
+
+describe('normalizeText()', () => {
+    test('empty text', () => {
+        expect(normalizeText('')).toBeUndefined()
+    })
+
+    test('normalized text', () => {
```

Comment:
It's better to understand, what it does, when you describe it in test case description.
For example:
```suggestion
    it('returns normalized text as is', () => {
```